views/peeps/index.erb

```diff
@@ -0,0 +1,25 @@
+<h1>Chitter</h1>
+
+<h2>
+  <%= flash[:notice] %>
+</h2>
+
+<% if @maker %>
+  <h2>Welcome <%= @maker.username %></h2>
+<% end %>
+
+<ul>
+  <% @peeps.reverse_each do |peep| %>
```

Comment:
Could put the reversing logic in the controller instead of the view (i.e using ORDER BY in the SQL query)